src/Tester/ManagementGrainTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Orleans;
+using Orleans.Runtime;
+using UnitTests.GrainInterfaces;
+using UnitTests.Grains;
+using UnitTests.Tester;
+
+// ReSharper disable ConvertToConstant.Local
+
+namespace UnitTests.Management
+{
+    [TestClass]
+    public class ManagementGrainTests : UnitTestSiloHost
```

Comment:
Is this class being moved or what? Because it shows as an add. I'm just asking, as I have updated this class in a non-trivial way already in a local branch.